doc/classes/Plane.xml

```diff
@@ -147,36 +140,36 @@
 			<argument index="0" name="point" type="Vector3">
 			</argument>
 			<description>
-				Returns the orthogonal projection of point [code]p[/code] into a point in the plane.
+				Returns the orthogonal projection of [code]point[/code] into a point in the plane.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="d" type="float" setter="" getter="" default="0.0">
-			Distance from the origin to the plane, in the direction of [member normal].
+			The distance from the origin to the plane, in the direction of [member normal]. This value is typically non-negative.
 		</member>
 		<member name="normal" type="Vector3" setter="" getter="" default="Vector3( 0, 0, 0 )">
-			The normal of the plane. "Over" or "Above" the plane is considered the side of the plane towards where the normal is pointing.
+			The normal of the plane (in the plane equation: a, b, and c). The normal vector must be normalized.
 		</member>
 		<member name="x" type="float" setter="" getter="" default="0.0">
-			The [member normal]'s X component.
+			The X component of the plane's [member normal] vector.
```

Comment:
I wonder if we should rename `x`, `y` and `z` to `a`, `b` and `c` to further the equivalence with the scalar plane equation?

Reply:
Maybe, either way it's clear to me. If so, it should be done in a future PR, and before 4.0.